src/common/plugins/plugins-config.js

```diff
@@ -149,6 +175,7 @@ function getEncodedReferrer(): string {
  */
 function _formatConfigString(config: string): Object {
   let configObj;
+  config = _sanitize(config);
```

Comment:
@lianbenjamin seems need to check here if config is not empty then you do not need the validation there 
and avoid parsing empty stuff

Reply:
I check inside _sanitize method